apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -72,6 +72,9 @@ class OrderBlocksAnswerData(TypedDict):
 FIRST_WRONG_TYPES = frozenset(
     [FeedbackType.FIRST_WRONG, FeedbackType.FIRST_WRONG_VERBOSE]
 )
+LCS_GRADABLE_TYPE = frozenset(
```

Comment:
should be 'TYPES' like the one above it, not 'TYPE'

Reply:
Fixed [2e87c5b](https://github.com/PrairieLearn/PrairieLearn/pull/9292/commits/2e87c5b944528607baf0bcc04c1665f93bc9315d)